lib/pynml/spec.py

```diff
@@ -590,6 +590,20 @@
                 }
             ]
         },
+        {
+            'name': 'Environment',
+            'parent': None,
+            'brief': 'Describes attributes inherent to the environment',
+            'doc': (
+                'Attributes to be attached to the environment the topology is'
+                'in.'
+            ),
+            'abstract': False,
+            'attributes': [
+            ],
+            'relations': [
```

Comment:
An environment should have a relation to a topology. Based off "2.2.7 hasNode" please create a new relation hasTopology that bounds 1 environment to N topologies.

Reply:
Done, but instead Topology has a relationship hasEnvironment